ERCS/erc-7704.md

```diff
@@ -0,0 +1,172 @@
+---
+eip: 7704
+title: Soul Resonance Token
+description: this EIP defines an interface for Soul Resonance Tokens
+author: Mike Smith (@mike0x1024), Abe Johnson <abe@burve.io>, Alvis Kaiser <alvis@burve.io>, WurDst Tsao <wurst@burve.io>
+discussions-to: https://ethereum-magicians.org/t/eip-7704-soul-resonance-token/20058
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-04-28
+requires: 20
+---
+
+## Abstract
+
+This EIP proposes the following improvements to Soul Bond Tokens:
+Extensibility to [ERC-20](./eip-20.md).
+Token Minting and Burning while preserving non-transferability.
+
+## Motivation
+
+The inspiration behind this EIP stems from discussions surrounding the advantages and potential applications of Soulbound/Account Bound Tokens (SBTs). However, upon analysing the pros and cons of SBTs, it has been determined that while the non-transferable nature of SBTs, which binds them to user accounts, can serve as a notable feature akin to a badge of honour, it also impedes the fluidity and evolution of the tokens themselves. SRT, short for Soul Resonance Token, is a special token protocol. Like SBT, SRT does not support transfer methods, meaning users cannot transfer SRT assets between different accounts. What makes SRT special is that, through the Bonding Curve protocol, the mint() and burn() methods are constructed to achieve pricing and trading functions for SRT. Therefore, SRT is a type of asset that cannot be transferred but can be traded. SRT fills the gap between SBT and traditional [ERC-20](./eip-20.md) assets, possessing both the soul-binding characteristics of SBT and the tradability of traditional [ERC-20](./eip-20.md) assets.
+
+## Specification
+
+Notes:
+The contract is designed to be compiled with a Solidity compiler version greater than or equal to 0.8.20 but less than 0.9.0.
+
+The smart contract realising this EIP mandate must fully incorporate all methods prescribed by [ERC-20](./eip-20.md), with the exception of transfer operations. Furthermore, exclusivity to implementing solely this EIP is required.
```

Comment:
> transfer operations

Be explicit. Which transfer operations?

> Furthermore, exclusivity to implementing solely this EIP is required.

As respectfully as possible, this is a terrible requirement. ERC-165 is a great addition to this proposal. I could see ERC-173 or ERC-5679 also being useful.